ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -299,10 +299,52 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("repeat") {
-			pending("should start a signal N times upon completion") {
+			it("should start a signal N times upon completion") {
+				let original = SignalProducer<Int, NoError>(values: [1, 2, 3])
+				let producer = original |> repeat(2)
+
+				let result = producer |> allValues
+				expect(result.value).to(equal([1, 2, 3, 1, 2, 3, 1, 2, 3]))
 			}
 
-			pending("should not repeat upon error") {
+			it("should produce an equivalent signal producer if count is 0") {
+				let original = SignalProducer<Int, NoError>(value: 1)
+				let producer = original |> repeat(0)
+
+				let result = producer |> allValues
+				expect(result.value).to(equal([1]))
+			}
+
+			it("should not repeat upon error") {
+				let results: [Result<Int, TestError>] = [
+					success(1),
+					success(2),
+					failure(.Default)
+				]
+
+				var operationIndex = 0
+				let operation: () -> Result<Int, TestError> = {
+					assert(operationIndex < results.count, "Operation started too many times")
```

Comment:
Is there an equivalent to `except` that actually marks the test case as a failure and stops? If I just replace this with `except` the next line would be an out of bounds runtime crash.